### PR TITLE
Create PR for dependency updates

### PR DESCRIPTION
**Pull request by TharindaDilshan**
Creating a PR instead of directly committing dependency updates

### CODE REVIEW

File: .github/workflows/dependency_graph.yml [hunk 1]
```diff
@@ -12,6 +12,21 @@ jobs:
...
         steps:
             -   uses: actions/checkout@v2
+
+            -   name : Configure GitHub and create branch
+                env:
+                    GITHUB_TOKEN: ${{ secrets.BALLERINA_BOT_TOKEN }}
+                run: |
+                    git config user.name ${{ secrets.BALLERINA_BOT_USERNAME }}
+                    git config user.email ${{ secrets.BALLERINA_BOT_EMAIL }}
+                    echo "BRANCH_EXISTS=0" >> $GITHUB_ENV
```
Comment (ThisaruGuruge) [inline .github/workflows/dependency_graph.yml:22]: Can't we directly set this as an environment variable similar to line no. 18?

File: .github/workflows/dependency_graph.yml [hunk 2]
```diff
@@ -28,21 +43,21 @@ jobs:
...
+                        git remote set-url origin https://${{ secrets.BALLERINA_BOT_USERNAME }}:${{ secrets.BALLERINA_BOT_TOKEN }}@github.com/ballerina-platform/ballerina-standard-library.git
+                        git add .
+                        git commit -m "stdlib_modules.json updated" || echo "No changes to commit"
+                        git push origin dependency-graph
+                        if hub pr list | grep 'StdLib Module Dependencies Updated'; then    
+                            echo "PR already exists"
+                        else
+                            echo "Creating PR"
+                            curl -fsSL https://github.com/github/hub/raw/master/script/get | bash -s 2.14.1
+                            bin/hub pull-request -m "StdLib Module Dependencies Updated"
```
Comment (ThisaruGuruge) [inline .github/workflows/dependency_graph.yml:61]:
```suggestion
                            bin/hub pull-request -m "[Automated] Update stdlib module dependencies"
```

File: .github/workflows/dependency_graph.yml (hunk 1 above)
Comment (TharindaDilshan) [reply .github/workflows/dependency_graph.yml:22]: The value is being updated dynamically.

Comment (ThisaruGuruge) [reply .github/workflows/dependency_graph.yml:22]: Still, we need to define it first right? This line is just defining the `BRANCH_EXISTS` variable with the initial value of `0`.

Comment (TharindaDilshan) [reply .github/workflows/dependency_graph.yml]: Updated

**Review by ThisaruGuruge — APPROVED** [review]

**Pull request merged**